src/Badges.sol

```diff
@@ -116,26 +116,26 @@ contract Badges is
       super.supportsInterface(interfaceId);
   }
 
-  function name() public view virtual override returns (string memory) {
+  function name() external view virtual override returns (string memory) {
     return _name;
   }
 
-  function symbol() public view virtual override returns (string memory) {
+  function symbol() external view virtual override returns (string memory) {
     return _symbol;
   }
 
-  function tokenURI(uint256 tokenId) public view virtual override returns (string memory) {
+  function tokenURI(uint256 tokenId) external view virtual override returns (string memory) {
     require(_exists(tokenId), "tokenURI: token doesn't exist");
     return _tokenURIs[tokenId];
   }
 
-  function unequip(uint256 tokenId) public virtual override {
+  function unequip(uint256 tokenId) external virtual override {
```

Comment:
Hmm.. I think some of these functions if marked as external, will not be available for other contracts to call them, no?

Reply:
Other contracts could call it but if another contract inherits from this, that contract would not be able to call it internally. You're right, I'll change these back. We should leave that flexibility.